run/rails-cat_album/app/javascript/channels/consumer.js

```diff
@@ -0,0 +1,6 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
```

Comment:
Channels are ActionCable related as well. So remove these, and the import in application.js